frontend/src/styles/colors.scss

```diff
@@ -1,29 +1,35 @@
-$main-color: #fff;
-$background-color: #3f51b5;
-$background-color-dark: #32408f;
-$delete-color: #e62116;
-$danger-background-color: #ff9999;
-$good-color: #028802;
-$good-color-light: #90ee90;
-$shadow-box: #00000042;
-$thunck-color: #3333336c;
-$thunck-color-hover: #333333b9;
-$group-background-color: #edefff;
-$light-blue: #add8e6;
-$border-color: #000000;
-$label-color: #9c9c9c;
-
-
-$dark-blue: #3f51b5;
+$red: #e62116;
 $selected-blue: #0000001a;
 $black: #0f0f0f;
 $gray: #6b7280;
 $light-gray: #d1d5db;
 $light-clear-gray: #edeef1;
 $snowy-white: #ffffff;
+$yellow: #f7f7c1;
 $blue: #1976d2;
 $blue-hover: #155da5;
+$light-blue: #4c94f6;
 $dark-blue: #3f51b5;
 $selected-blue: #0000001a;
-$light-forest: #028802;
-$forest: #184b44;
+$green: #028802;
+$dark-green: #184b44;
+
+$create-color-light: #90ee90;
+$create-color: $green;
+$edit-color: $blue;
+$delete-color: $red;
+$warning-color: $yellow;
```

Comment:
as above
this file need to refactor

Reply:
I've created something like constants for certain events for better scalability in the future